docs/reference/api/docker_remote_api_v1.21.md

```diff
@@ -2603,39 +2655,44 @@ Status Codes:
 
 **Example request**:
 
-  GET /networks/f995e41e471c833266786a64df584fbe4dc654ac99f63a4ee7495842aa093fc4 HTTP/1.1
+    GET /networks/f2de39df4171b0dc801e8002d1d999b77256983dfc63041c0f34030aa3977566 HTTP/1.1
 
 **Example response**:
 
-  HTTP/1.1 200 OK
-  Content-Type: application/json
-
 ```
-  {
-    "name": "bridge",
-    "id": "f995e41e471c833266786a64df584fbe4dc654ac99f63a4ee7495842aa093fc4",
-    "driver": "bridge",
-    "containers": {
-      "931d29e96e63022a3691f55ca18b28600239acf53878451975f77054b05ba559": {
-        "endpoint": "aa79321e2899e6d72fcd46e6a4ad7f81ab9a19c3b06e384ef4ce51fea35827f9",
-        "mac_address": "02:42:ac:11:00:04",
-        "ipv4_address": "172.17.0.4/16",
-        "ipv6_address": ""
-      },
-      "961249b4ae6c764b11eed923e8463c102689111fffd933627b2e7e359c7d0f7c": {
-        "endpoint": "4f62c5aea6b9a70512210be7db976bd4ec2cdba47125e4fe514d18c81b1624b1",
-        "mac_address": "02:42:ac:11:00:02",
-        "ipv4_address": "172.17.0.2/16",
-        "ipv6_address": ""
-      },
-      "9f6e0fec4449f42a173ed85be96dc2253b6719edd850d8169bc31bdc45db675c": {
-        "endpoint": "352b512a5bccdfc77d16c2c04d04408e718f879a16f9ce3913a4733139e4f98d",
-        "mac_address": "02:42:ac:11:00:03",
-        "ipv4_address": "172.17.0.3/16",
-        "ipv6_address": ""
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+  "Name": "bridge",
+  "Id": "f2de39df4171b0dc801e8002d1d999b77256983dfc63041c0f34030aa3977566",
+  "Scope": "local",
+  "Driver": "bridge",
+  "IPAM": {
+    "Driver": "default",
+    "Config": [
+      {
+        "Subnet": "172.17.0.0/16"
       }
+    ]
+  },
+  "Containers": {
+    "39b69226f9d79f5634485fb236a23b2fe4e96a0a94128390a7fbbcc167065867": {
+      "EndpointID": "ed2419a97c1d9954d05b46e462e7002ea552f216e9b136b80a7db8d98b442eda",
+      "MacAddress": "02:42:ac:11:00:02",
+      "IPv4Address": "172.17.0.2/16",
+      "IPv6Address": ""
     }
+  },
+  "Options": {
+    "com.docker.network.bridge.default_bridge": "true",
```

Comment:
these options appeared in the output, but wasn't sure these should be included in the documentation

@mavenugo ?